src/main/scala/org/constellation/gossip/snapshot/SignedSnapshotProposalGossip.scala

```diff
@@ -0,0 +1,18 @@
+package org.constellation.gossip.snapshot
+
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder}
+import org.constellation.schema.Id
+import org.constellation.schema.signature.HashSignature
+
+import scala.collection.SortedMap
+
+case class SignedSnapshotProposalGossip(hash: String,
```

Comment:
Please format it using scalafmt. You can setup it in your IDE

Reply:
Thanks @marcinwadon I thought I had it configured.